src/asm/macro.c

```diff
@@ -117,7 +117,7 @@ void macro_SetUniqueID(uint32_t id)
 		if (uniqueID > maxUniqueID)
 			maxUniqueID = uniqueID;
 		/* The buffer is guaranteed to be the correct size */
-		sprintf(uniqueIDBuf, "_%" PRIu32, id);
+		sprintf(uniqueIDBuf, "@%" PRIu32, id);
```

Comment:
I'd prefer to avoid using `@` (and `#`) in symbol names, as it may be useful to eventually change their rules to give them special meaning. For example, there was talk to implement #342 using symbols prefixed with `@`.
Not sure what would be a better alternative, though?

Reply:
I only changed that so things like `ld a, 4\@` wouldn't be unintentionally valid. For that purpose, even a format like `"_Q%" PRIu32` would be OK; it just seems more arbitrary than `"_%"` or `"@%"`.

Reply:
Yes, this is why I said I wasn't sure about a better alternative. Maybe `"_u%" PRIu32`, with "u" for "unique"?